android-kotlin/app/src/main/java/com/mongodb/app/MyApp.kt

```diff
@@ -0,0 +1,39 @@
+package com.mongodb.app
+
+import android.app.Application
+import android.util.Log
+
+import io.realm.Realm
+import io.realm.log.LogLevel
+import io.realm.log.RealmLog
+import io.realm.mongodb.App
+import io.realm.mongodb.AppConfiguration
+
+lateinit var myApp: App
```

Comment:
Minor: I would actually rename this to `realmApp`. `my`... is great for tutorials, but in a real app I wouldn't expect to see it so much.